ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml

```diff
@@ -0,0 +1,30 @@
+binary_build_commands: go install ./
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
```

Comment:
Looks like golang-1.20 image stream is not yet available. There is a [issue](https://redhat-internal.slack.com/archives/CBN38N3MW/p1687379872277429) reported for the same in the forum-testplatform channel.

Reply:
I have switched it to 1.19